setup.py

```diff
@@ -33,9 +36,10 @@ def _plain(text: str) -> str: return text
     bright = dim = red = green = yellow = _plain
 
 ROOT = Path(__file__).resolve().parent
+SRC_ROOT = ROOT / "src"
 BUILD_JS = ROOT / 'bokehjs' / 'build' / 'js'
 BUILD_TSLIB = ROOT / 'bokehjs' / 'node_modules' / 'typescript' / 'lib'
-PKG_STATIC = ROOT / 'src' / 'bokeh' / 'server' / 'static'
+PKG_STATIC = SRC_ROOT / 'bokeh' / 'server' / 'static'
```

Comment:
@mattpap this `ROOT` / `SRC_ROOT` business raises the inconsistency of having `bokehjs` outside the `src` dir. Would you want to consider moving it there? With src layout, it would be trivial to exclude a `src/bokehjs` dir from the  python package, being source code, technically under the `src` dir would be the most sensible place for `bokehjs` to live.

Reply:
I do genuinely think this is worth discussing/considering for the sake of  getting the repo structure into a complete clean and consistent structure. Do let me know what you think.

Reply:
I'm aware, I just need to find time to think about this. Perhaps you could turn that comment into a discussion, so that this isn't forgotten and there's a place to discuss this.